x-pack/plugins/advanced_ui_actions/public/can_inherit_time_range.test.ts

```diff
@@ -5,10 +5,8 @@
  */
 import { canInheritTimeRange } from './can_inherit_time_range';
 /** eslint-disable */
-import {
-  HelloWorldEmbeddable,
-  HelloWorldContainer,
-} from '../../../../src/plugins/embeddable/public/lib/test_samples';
+import { HelloWorldContainer } from '../../../../src/plugins/embeddable/public/lib/test_samples';
```

Comment:
yea, there is a `/* eslint-disable */` surrounding it (as there was originally).  The example plugins actually make this easier because they can export the examples at the top level, while we don't want that in here.  I've thought about replacing all these samples with ones from the examples repo but some really are only built for testing not so much examples.  Some of this could still probably be cleaned up and more example embeddables used for testing.